packages/gatsby-theme-newrelic/README.md

```diff
@@ -68,17 +67,16 @@ websites](https://opensource.newrelic.com).
     - [`NavItem`](#navitem)
       - [`Page`](#page)
     - [`NewRelicLogo`](#newreliclogo)
-    - [`Overlay`](#overlay)
+    - [Overlay](#overlay)
     - [`PageTools`](#pagetools)
       - [`PageTools.Section`](#pagetoolssection)
       - [`PageTools.Title`](#pagetoolstitle)
-      - [`Portal](#portal)
+    - [Portal](#portal)
```

Comment:
same question as above

Reply:
Oh that's a good shout @josephgregoryii they should be present as the other table of contents items have them 😸